### ISSUE TITLE
Problem with oneSvgPerLine

### ISSUE THREAD

**Issue by Jannie62** [paulrosen/abcjs#244]
Dear Paul and other developers of abcjs,

Thank you for abcjs!

I'am going to build a church-presentation website (kerkmemo.nl) with a database for presentation of song-lyrics and melody line on screens/beamers during services.
Your libary is very useful for me!

Because max-lyric-text-size is to small for my neads (spaces become to small because text is aligned under notes) I want to place my own left-aligned lyric-text between the lines.
Because of that, I added 'oneSvgPerLine' in parameterlist:
 
new ABCJS.Editor("abc", { canvas_id: "paper",
				warnings_id: "warnings",
				abcjsParams: {responsive: "resize",
					add_classes: true,
					oneSvgPerLine: true,}
			});

This doesn't work. I still have one SVG-div for the whole song.
See my adapted editor.html at [https://www.kerkmemo.nl/kerkmemo/abcjs-master/examples/editor.html]

Maybe I do something wrong or is it a bug?

Thanks in advance!
Jannie

**Comment by paulrosen** [comment]
I think oneSvgPerLine isn't compatible with the Editor - it should be, though, so that's a bug.

Are you depending on the editor or will you be using `abcjs.renderAbc()` because you will be feeding known melodies to your page? In any case, try the similar parameters using the `basic.html` version.

There may be other ways to do what you want, though, too. Try this string in the editor:
```
X:1
T: Cooley's
M: 4/4
L: 1/8
R: reel
K: Em
|:D2|EB{c}BA B2 EB|~B2 AB dBAG|FDAD BDAD|FDAD dAFD|
%%text Deck the halls
EBBA B2 EB|B2 AB defg|afe^c dBAF|DEFD E2:|
%%text Tra la la la la
|:gf|eB B2 efge|eB B2 gedB|A2 FA DAFA|A2 FA defg|
%%text la la la la
eB B2 eBgB|eB B2 defg|afe^c dBAF|DEFD E2:|
```
If you add classes to that, then you can style that text.

**Comment by Jannie62** [comment]
Hi Paul,
Thank you for rapid reply!

- In presentation part of my software I only need to render existing abc-notation, but for editors it must by possible to add/edit abc-notation of a song. Presenter-mode need big fonts but editor-mode doesn't.
- I tested in basic.html with same result as in editor.html. Still one svg.
code: 
new ABCJS.Editor("abc", { canvas_id: "paper",
				warnings_id: "warnings",
				abcjsParams: `{responsive:` "resize",
					add_classes: true,
					oneSvgPerLine: true,}
			});
- Your second suggestion with '%%text' works fine, but I don't know how I can add classes to this text because I can't find %%text in documentation of abcjs. Can you explain this to me?

Thanks in advance,
Jannie

**Comment by paulrosen** [comment]
`%%text` is part of the abc standard, not necessarily part of abcjs. In any case, what I was suggesting is to take the `basic.html` example and replace the javascript with this:
```
<script type="text/javascript">
	var abc = "T: Cooley's\n" +
		"M: 4/4\n" +
		"L: 1/8\n" +
		"R: reel\n" +
		"K: Emin\n" +
		"|:D2|EB{c}BA B2 EB|~B2 AB dBAG|FDAD BDAD|FDAD dAFD|\n" +
		"%%text Tra la la la la\n" +
		"EBBA B2 EB|B2 AB defg|afe^c dBAF|DEFD E2:|\n" +
		"%%text Tra la la la la\n" +
		"|:gf|eB B2 efge|eB B2 gedB|A2 FA DAFA|A2 FA defg|\n" +
		"%%text Tra la la la la\n" +
		"eB B2 eBgB|eB B2 defg|afe^c dBAF|DEFD E2:|\n" +
		"%%text Tra la la la la\n";
		function load() {
		ABCJS.renderAbc("paper", abc, {
			responsive: "resize",
			add_classes: true,
			format: {
				textfont: "cursive 40"
			}
		});
	}
</script>
```

I just tried it, though, and I see there are some spacing issues - that's a bug and I'll get on it really soon. You should be able to control the family and size of the text with the `textfont` line.

From what you describe, if I were drawing the text in the right place, I think it should work for your use case.

**Comment by Jannie62** [comment]
Hi Paul,

Thank you again for rapid reply!

After reading your answer, I saw that my second copy was made of a wrong html-file. I placed a second copy of my modified editor.html instead of my modified basic.html
Sorry!

After removing spacing issues your last code will work voor me.
Thank you for helping me!

Jannie

**Comment by paulrosen** [comment]
Will be fixed in 5.8.1
